crates/sargon/src/signing/collector/signatures_collector.rs

```diff
@@ -95,12 +95,12 @@ impl SignaturesCollector {
         let transactions = transactions
             .into_iter()
             .map(extract_signers)
-            .collect::<Result<IndexSet<TXToSign>>>()?;
+            .collect::<Result<Vec<TXToSign>>>()?;
```

Comment:
did you get a compilation error when using  `.collect::<Result< IdentifiedVecOf <TXToSign>>>()?` (directly, instead of via `Vec`)?

Reply:
No just dumb me.